source/MRMesh/MRTriMesh.h

```diff
@@ -1,6 +1,8 @@
 #pragma once
 
 #include "MRVector.h"
+#include "MRId.h"
+#include "MRVector3.h"
```

Comment:
Can it be avoided?

Reply:
This file won't compile without this include anyway.